k6script.js

```diff
@@ -0,0 +1,37 @@
+import { check, sleep } from 'k6';
+import http from 'k6/http';
+import idStrings from './k6RandomIds.js';
+
+export const options = {
+  stages: [
+    { duration: '2m', target: 1 }, // below normal load
+    { duration: '5m', target: 1 },
+    { duration: '2m', target: 10 }, // normal load
+    { duration: '5m', target: 10 },
```

Comment:
hey just curious: why break these into 2m & 5m blocks rather than one 7m?  are you waiting for them to resolve all requests in between?